imjoy/workers/worker_template.py

```diff
@@ -64,7 +70,7 @@ class PluginConnection:
 
     # pylint:disable=too-many-instance-attributes
 
-    def __init__(self, opt):
+    def __init__(self, opt, client=None):
```

Comment:
When do we need to pass a client?

Reply:
This will be used in the tests where the `fake_client.test_client` will be passed in.